### PR TITLE
add files precourse

### PR DESCRIPTION
**Pull request by manoli2013**

### CODE REVIEW

File: staff/noli-pascual/precourse/connect-4/style.css [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+  background: #fff;
+  border: 2px solid #073976;
+  width: 25px;
+  height: 25px;
+  text-align: center;
+  color: #ffffff;
+
+}
+
+/* #results {
```
Comment (Fme82) [inline staff/noli-pascual/precourse/connect-4/style.css]: I think you forgot to delete this code (lines 136 to 147)😀

Comment (manoli2013) [reply staff/noli-pascual/precourse/connect-4/style.css]: I can't find the lines!

**Review by manoli2013 — COMMENTED** [review]
modified

**Comment by manoli2013** [comment]
modified

**Pull request merged**